### DISCUSSION TITLE
Setting time format on UI

### DISCUSSION THREAD

**Discussion by MisterTdog** [blakeblackshear/frigate#2629] [Ask A Question] [answered]
Hi,
Is there a way to set up the time format on the UI?
ex: in Event page, I see: 1/12/2022 3:33:12 PM
I would prefer have: 12/01/2022 15:33:12

**Comment by krzyk** [comment]
Exactly I can't find setting for that and both my server and my browser have correct locale set.

**Comment by MarkusMad** [comment]
For me if I change my browsers language setting the time format in frigate gets displayed correctly - 24h instead of 12h AM/PM setting, however I would prefer my browser to stay on en_US and nevertheless have frigate show time in a 24h format. 

The docker container's time is also using a 24h format, but this hasn't anything to do with it as far as I can tell. 

Perhaps as additional option in "frigate.yml" would be able to solve this.

**Comment by NickM-27** [comment, marked as answer]
This has been implemented in https://github.com/blakeblackshear/frigate/pull/3309

**Reply by MarkusMad** [reply]
Thanks, highly appreciated!
